irc/quassel/Portfile

```diff
@@ -35,9 +35,19 @@ configure.args  -DWANT_CORE=OFF \
                 -DWANT_QTCLIENT=OFF \
                 -DWANT_MONO=OFF \
                 -DWITH_KDE=OFF \
+                -DWITH_WEBENGINE=OFF \
                 -DUSE_CCACHE=OFF \
                 -DUSE_QT5=ON
 
+variant urlpreview description "Support for showing URL thumbnails" {
+    qt5.depends_component qtwebengine
+    configure.args-replace -DWITH_WEBENGINE=OFF -DWITH_WEBENGINE=ON
+}
+
+if {${os.major} < 21} {
+    default_variants +urlpreview
```

Comment:
you're changing default variants so I *think* you would need to increase the `revision` although I do agree that it doesn't change the installed files